### PR TITLE
Add module partner_contact_disability.

### PR DESCRIPTION
**Pull request by yajo**
Aiming to fix https://github.com/OCA/l10n-spain/issues/174, this module lets the user set a percentage of disability for any contact.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I think these 3 PRs adding a field should be joined in one _partner_contact_special_info_ or similar. What do you think?

**Comment by yajo** [comment]
I think I'm more comfortable splitting it. Maybe someone needs all of them for https://github.com/OCA/l10n-spain/issues/174, or maybe someone needs only one of them... I prefer to let admins choose.

File: partner_contact_disability/README.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Module name
```
Comment (yvaucher) [inline partner_contact_disability/README.rst]: This is a placeholder for the name of the module

**Comment by yajo** [comment]
Closing. See https://github.com/OCA/l10n-spain/issues/174#issuecomment-127557716.

**Pull request closed without merging**